src/mz/tests/e2e.rs

```diff
@@ -77,14 +77,35 @@
 
 #[cfg(test)]
 mod tests {
-    use std::{fs, io::Read, path::Path, time::Duration};
+    use std::{
+        fs,
+        io::Read,
+        path::{Path, PathBuf},
+        time::Duration,
+    };
 
     use assert_cmd::{assert::Assert, Command};
     use mz::{config_file::ConfigFile, ui::OptionalStr};
     use mz_frontegg_auth::AppPassword;
     use serde::{Deserialize, Serialize};
     use tabled::{Style, Table, Tabled};
 
+    fn get_config_path() -> PathBuf {
+        let config_file_path = ConfigFile::default_path().unwrap();
```

Comment:
I wonder if we actually just want to put this in a temp directory so it doesn't pollute your actual config dir? I don't have a strong opinion here, but that seems better to me.

Reply:
I agree!